### ISSUE TITLE
Error: Running LUA method 'update'. base.lua:1247: attempt to index local 'currentHelper' (a nil value)

### ISSUE THREAD

**Issue by RenanDup** [Courseplay/courseplay#7428]
Hi,
It's seems that it remains impossible to launch more than N tools simultaneously...

2021-08-14 07:55 C:/Users/XXX2/Documents/My Games/FarmingSimulator2019/mods/FS19_Courseplay/base.lua:1247: attempt to index local 'currentHelper' (a nil value)
2021-08-14 07:56 Error: Running LUA method 'update'.
2021-08-14 07:56 dataS/scripts/vehicles/Vehicle.lua(2920) : attempt to index a nil value
2021-08-14 07:56 Error: Running LUA method 'update'.

Do you have a idea?
Bye!

**Comment by Tensuko** [comment]
The best Idea for you is to read this first!
https://github.com/Courseplay/courseplay/wiki

**Comment by RenanDup** [comment]
I'm sorry, I'm not fluent... and I don't know very well how I have to do... So the log file is attached.
If you need something more, please, ask me.
Thank you.
[log.txt](https://github.com/Courseplay/courseplay/files/6987092/log.txt)

**Comment by Tensuko** [comment]
That is strange, we tested it with 12 Helper and it was more then the reported 10 maximum so we said ok that should work...
Will have a look...

**Comment by RenanDup** [comment]
I don't think the problem is 10, it occurs at different times.
In my last tests i had the impression that this happens when there was an interaction with AutoDrive... But I'm not sure.
I went back to the previous version which apparently works better for me ...

**Comment by KenMerritt** [comment]
I just want to say that this issue still exists. I can't say exactly how to replicate it, but here is what happened. I started two combines on courseplay. I then started 6 trucks using courseplay but using autodrive to drive to the sell point. Then when I tried to start another courseplay vehicle it gave the error message and wouldn't start. If I stopped one of the autodrive trucks, I was then able to start the couresplay vehicle and then I could go back and start the autodrive truck again. So there is still a limit but it only shows up depending on the order you start courseplay and autodrive vehicles in.
